### ISSUE TITLE
Multiplayer keep speed is bugged

### ISSUE THREAD

**Issue by Geo30033** [Mogli12/VehicleControlAddon#854]
When i play with my friend the keep speed even if its not activated it still works, we stop and it still accelerates. Hope i can hear a fix soon because it sucks and i would like to use the other features but this one is the worst on my opinion and destroys the gameplay. If you want to stop you go into a complete stop and when you let go of the brakes it still goes after but slow.

**Comment by Geo30033** [comment]
https://imgur.com/a/s9Kdyhk

**Comment by Mogli12** [comment]
Hi,
Is this really keep speed? If motor modifications are active then the vehicle will maintain speed at idle RPM.
Regards, Stefan

**Comment by Geo30033** [comment]
> Hi, Is this really keep speed? If motor modifications are active then the vehicle will maintain speed at idle RPM. Regards, Stefan

Yeah, it is, because if i hold brake it stops, and it should be stopped like parking brake if i remove brake but it doesnt, so if i want it to fully stop i need to hold delete then brake, because sometimes its 10kmh 8kmh randomly and holds it, even in reverse. I have this problem when i play on multiplayer with a friend but on singleplayer i dont have it

**Comment by Mogli12** [comment]
Do you use shuttle control? Did you break and stop for at least 2 seconds?

**Comment by Geo30033** [comment]
> Do you use shuttle control? Did you break and stop for at least 2 seconds?

I dont know what shuttle control is, and yeah i did break and stop for at least 2 seconds, check the video on imgur

**Comment by Mogli12** [comment]
For me it looks like the speed was 0 was less than 2 seconds only.

**Comment by Geo30033** [comment]
> For me it looks like the speed was 0 was less than 2 seconds only.

Near the finish of the video i held more, but you get the point, it just keeps going and we didnt activate keep speed, but if i hold delete then brake, it stops and stays there. So i know its from keep speed

**Comment by Mogli12** [comment]
Please check if the problem is fixed with build 119
